res/database/Default/user/kasi.xml

```diff
@@ -155,7 +155,7 @@
 		</scripttemplate>
 
 		<!--Besetzung Musiker, Livedatum, nur eine Ausstrahlung, Produktionszeit (Vorproduktion), kein optionaler Kult-->
-		<scripttemplate licence_type="1" product="3" guid="Kasi-Script-LiveKonzert">
+		<scripttemplate licence_type="1" product="3" guid="Kasi-Script-LiveKonzert" comment="TODO: Though it doesn't matter much in effect, it's confusing and not clear for (new) players why you have to cast a musician when there's already a musician in the title. Support act? Co-musician? If so, that should be added to the description. Or should there be a feature request to have some sort of pre-selected cast (which can't be changed in this example)?">
```

Comment:
Somewhere on my TODO-Lists, I have the feature request, that a cast can be preselected by a variable (e.g. list of possible main actors as variable, and you have to live with the one randomly selected).
In this instance, I interpret the musician as support act - to raise the price of the production.

Reply:
Is there a link to the feature request / issue? If not I'll make one.

Reply:
I don't think there is an explicit ticket. This idea is related to extending the variable resolution in general and for script templates in particular. Feel free to open a ticket. But the new variable syntax is a precondition for addressing this.

Reply:
https://github.com/TVTower/TVTower/issues/1065